Add gcc-multilib for building for Raspberry Pi 4 B as target

Hey,
first of all, thanks for this. Whenever I have to compile the kernel or parts of any project, I always have to jump through multiple hoops in order to get the build process to work only to notice that after an hour it still fails. This image helped me so much to speed up the process.

I only have one issue when trying to build the v23.05.2 image for my Raspberry Pi 4B
Similar to [here](https://github.com/openwrt/openwrt/issues/12239), [here](https://github.com/openwrt/openwrt/issues/10201) and [here ](https://forum.openwrt.org/t/error-compiling-23-05-linksys-wrt3200acm/181024) I'm getting this error:

```
make[3]: Entering directory '/home/user/openwrt/package/network/utils/xdp-tools'
rm -f /home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9/.built
touch /home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9/.built_check
CFLAGS="-Os -pipe -fno-caller-saves -fno-plt -fhonour-copts -ffile-prefix-map=/home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9=xdp-tools-1.2.9 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro    -I/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/usr/include -I/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/include/fortify -I/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/include   " CXXFLAGS="-Os -pipe -fno-caller-saves -fno-plt -fhonour-copts -ffile-prefix-map=/home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9=xdp-tools-1.2.9 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro    -I/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/usr/include -I/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/include/fortify -I/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/include   " LDFLAGS="-L/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/usr/lib -L/home/user/openwrt/staging_dir/toolchain-aarch64_cortex-a72_gcc-12.3.0_musl/lib -znow -zrelro    " PREFIX=/usr RUNDIR=/tmp/run make  -C /home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9/. AR="aarch64-openwrt-linux-musl-gcc-ar" AS="aarch64-openwrt-linux-musl-gcc -c -Os -pipe -fno-caller-saves -fno-plt -fhonour-copts -ffile-prefix-map=/home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9=xdp-tools-1.2.9 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro  " LD=aarch64-openwrt-linux-musl-ld NM="aarch64-openwrt-linux-musl-gcc-nm" CC="aarch64-openwrt-linux-musl-gcc" GCC="aarch64-openwrt-linux-musl-gcc" CXX="aarch64-openwrt-linux-musl-g++" RANLIB="aarch64-openwrt-linux-musl-gcc-ranlib" STRIP=aarch64-openwrt-linux-musl-strip OBJCOPY=aarch64-openwrt-linux-musl-objcopy OBJDUMP=aarch64-openwrt-linux-musl-objdump SIZE=aarch64-openwrt-linux-musl-size CROSS="aarch64-openwrt-linux-musl-" ARCH="aarch64" ;
make[4]: Entering directory '/home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9'

lib

  libxdp
    CC       staticobjs/libxdp.o
    CC       staticobjs/xsk.o
    CLANG    xdp-dispatcher.o
In file included from xdp-dispatcher.c:3:
In file included from ../../headers/linux/bpf.h:11:
/usr/include/linux/types.h:5:10: fatal error: 'asm/types.h' file not found
#include <asm/types.h>
         ^~~~~~~~~~~~~
1 error generated.
make[6]: *** [Makefile:139: xdp-dispatcher.o] Error 1
make[5]: *** [Makefile:20: libxdp] Error 2
make[4]: *** [Makefile:25: lib] Error 2
make[4]: Leaving directory '/home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9'
make[3]: *** [Makefile:139: /home/user/openwrt/build_dir/target-aarch64_cortex-a72_musl/xdp-tools-1.2.9/.built] Error 2
make[3]: Leaving directory '/home/user/openwrt/package/network/utils/xdp-tools'
time: package/network/utils/xdp-tools/compile#2.35#0.27#2.55
    ERROR: package/network/utils/xdp-tools failed to build.
make[2]: *** [package/Makefile:128: package/network/utils/xdp-tools/compile] Error 1
make[2]: Leaving directory '/home/user/openwrt'
make[1]: *** [package/Makefile:122: /home/user/openwrt/staging_dir/target-aarch64_cortex-a72_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/user/openwrt'
make: *** [/home/user/openwrt/include/toplevel.mk:232: world] Error 2
```

The solution according to [this](https://github.com/xdp-project/xdp-tools/issues/4#issuecomment-800027556) comment is to install `gcc-multilib`.

Should I open a PR or is this image only intended for x86_64 targets?

--- a/Dockerfile
+++ b/Dockerfile
@@ -6,7 +6,7 @@ RUN apt-get update &&\
         libssl-dev patch libncurses5 libncurses5-dev zlib1g-dev gawk \
         flex gettext wget unzip xz-utils python python-distutils-extra \
         python3 python3-distutils-extra rsync curl libsnmp-dev liblzma-dev \
-        libpam0g-dev cpio rsync && \
+        libpam0g-dev cpio rsync gcc-multilib && \
     apt-get clean && \
     useradd -m user && \
     echo 'user ALL=NOPASSWD: ALL' > /etc/sudoers.d/user

